packages/devextreme/testing/testcafe/tests/editors/slider/slider.ts

```diff
@@ -9,7 +9,9 @@ fixture.disablePageReloads`Slider`
 test('Slider appearance', async (t) => {
   const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
-  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container' });
+  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container', theme: 'light' });
```

Comment:
We don't need it, it uses light color scheme by default
```suggestion
  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container' });
```